src/ipahealthcheck/meta/core.py

```diff
@@ -13,8 +14,8 @@
 from ipapython.version import VERSION, API_VERSION
 from ipaplatform.paths import paths
 
-if 'FIPS_MODE_SETUP' not in dir(paths):
-    paths.FIPS_MODE_SETUP = '/usr/bin/fips-mode-setup'
+if 'PROC_FIPS_ENABLED' not in dir(paths):
+    paths.PROC_FIPS_ENABLED = '/proc/sys/crypto/fips_enabled'
```

Comment:
PROC_FIPS_ENABLED is in ipaplatform/base.py so should be visible in any derived platform. One can override a path but not remove it so I don't think this part is necessary. It's good defensive coding though.

Reply:
Thanks for letting me know!